pegasus/helpers/analytics_constants.rb

```diff
@@ -24,6 +24,7 @@ module AnalyticsConstants
     PICK_PL_PAGE_VISITED_EVENT = 'Pick Your Professional Learning Page Visited'.freeze,
     SELF_PACED_PL_PAGE_VISITED_EVENT = 'Self-Paced Professional Learning Page Visited'.freeze,
     RP_LANDING_PAGE_VISITED_EVENT = 'Regional Partner Landing Page Visited'.freeze,
+    TEACH_PAGE_VISITED_EVENT = 'Teach Page Visited'.freeze,
     TEACHER_LOGIN_EVENT = 'Teacher Login'.freeze
```

Comment:
I see this event is being used on the `TeacherHomepage.jsx` page. How is that page now being used, and can this event be removed? (not for this PR)

Reply:
I think that's a question for @dmcavoy if we want the teacher homepage to also trigger that event? Or if that should be a separate event? Or if we just remove it

Reply:
TeacherHomepage is the logged in page which lists all the sections for teachers and what now. Nothing is happening to that page and should stay the same